src/update_engine/postinstall_runner_action.cc

```diff
@@ -18,6 +18,10 @@ namespace {
 const char kPostinstallScript[] = "/postinst";
 }
 
+PostinstallRunnerAction::PostinstallRunnerAction(
+    std::string app_id)
+    : app_id_(app_id) {}
```

Comment:
I just merged C++11 yesterday, so you can rebase or make that change as a follow up I suppose.